v2/emailpassword/common-customizations/signin-form/adding-custom-error-message.mdx

```diff
@@ -0,0 +1,62 @@
+---
+id: adding-custom-error-message
+title: Adding Custom Error Message
+hide_title: true
+---
+
+import FrontendSDKTabs from "/src/components/tabs/FrontendSDKTabs"
+import TabItem from '@theme/TabItem';
+
+# Adding Custom Error Message
```

Comment:
We should create one single doc that has the following:
- contents from https://supertokens.com/docs/emailpassword/common-customizations/signup-form/changing-field-labels
- Changing optional error message
- setting default values
- creating custom components

Basically one document that talks about all the props that each formField takes.